contracts/cw1155-base/src/state.rs

```diff
@@ -0,0 +1,23 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::{CanonicalAddr, Uint128};
+use cw1155::Expiration;
+use cw_storage_plus::{Item, Map};
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
```

Comment:
Where is this used? In query response?

Reply:
it's initially copied from cw721 and never used, removed now.